lib/api/route/events.dart

```diff
@@ -16,7 +16,7 @@ Future<InitialSnapshot> registerQueue(ApiConnection connection) {
       'notification_settings_null': true,
       'bulk_message_deletion': true,
       'user_avatar_url_field_optional': false, // TODO(#254): turn on
-      'stream_typing_notifications': false, // TODO implement
+      'stream_typing_notifications': true,
```

Comment:
This should get its own commit and own commit message, since it's changing what we get from the server (whereas the rest of the commit has a very different character, adding UI).

I think the effect is that it's just causing us to get more `TypingEvent`s, and our code already handles the new kind?

Reply:
Yes, that's correct.

Reply:
commit-structure nit: put this commit before the main commit, so that the main commit can be accurate in claiming to fix the issue #665

(otherwise, #665 isn't actually fully working as of that commit)